BeerDiplomaWebpage/src/app/auth/auth.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+
+@Component({
+  selector: 'app-auth',
+  templateUrl: './auth.component.html',
+  styleUrls: ['./auth.component.css']
+})
+export class AuthComponent implements OnInit {
+  authType: String = '';
+  title: String = '';
+  jebanyan: Boolean = true;
```

Comment:
what is this? xD

Reply:
It’s just testing value, and it supposed to be named „jebanyangular” xd